File: 26/solution.go
package _26

import "math"

func removeDuplicates(nums []int) int {
	val := -1 * int(math.Pow(10, 4))
	idx := 0
	for i := 0; i < len(nums); i++ {
		if nums[i] != val {
			nums[idx] = nums[i]
			val = nums[i]
			idx += 1
		}
	}
	return idx
}

File: 26/Solution.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int idx = 0;
        int val = -1 * (int) Math.pow(10, 4) - 1;
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] != val) {
                nums[idx] = nums[i];
                val = nums[i];
                idx++;
            }
        }
        return idx;
    }
}